src/pages/Account.vue

```diff
@@ -53,7 +58,10 @@
 
                 </div>
 
-                <button id="requestVoucherHistoryEmail" v-on:click="onRequestVoucherHistoryEmail">Email payment history</button>
+                <div class="cta-buttons">
+                    <button id="requestVoucherHistoryEmail" v-on:click="onRequestVoucherHistoryEmail">Email all payment history</button>
+                    <button v-on:click="onRequestSubmissionEmail" :disabled="selected">Email selected payment history</button>
```

Comment:
Was it agreed with Kat to have the 2 buttons instead of just giving them one or other option? I think it makes the interface more confusing.